### ISSUE TITLE
It is not an API

### ISSUE THREAD

**Issue by houshyar** [fega/wordreference-api#6]
**API calls** provide a structured way to access data directly from a service's backend, unlike **web scraping** which parses raw HTML content.
` result.word = $('h3.headerWord').text()`
Your approach is highly depended on the structure of the website's HTML and if they change their design you should rewrite your code.
Maybe it's better to change your project's name to prevent confusion.

**Comment by prince-thind** [comment]
Right, you can get rate-limited very easily. Plus word-reference explicitly disallows bots activity in its robots.txt. Have you found any official API by them by any chance?
